### ISSUE TITLE
Location problem

### ISSUE THREAD

**Issue by Maxtermax** [chunyenHuang/hummusRecipe#14]
When try to call a script that use this library from other script using require fail, it throw a Segmentation fault (core dumped), is only allow if both script are in the same folder.
Same thing happend using the terminal, example:

`/path/to/pdf-script/ $node generate-pdf.js` 
it work fine.

`/path/to/ $node /pdf-script/generate-pdf.js` 
a directory up, throw Segmentation fault (core dumped)

using require in node get the same problem.

**Comment by chunyenHuang** [comment]
Well, I dont really follow, and dont know how to reproduce your issue. Maybe you can post some sample codes?

I will also compile the source codes to a single file script, perhaps it will fix this problem.

**Comment by Maxtermax** [comment]
Hi, so sorry for get late, the bug can be reproduced in this example:

```javascript
// service/generatePdf.js
const HummusRecipe = require('hummus-recipe');
const PATH =  './output.pdf';//create the new pdf in this path 
const IMAGE = './image.jpg';//path to a image

function CreateSimplePdf() {
  const pdfDoc = new HummusRecipe('new', PATH, {
    version: 1.6,
    author: 'John Doe',
    title: 'Hummus Recipe',
    subject: 'A brand new PDF'
  });
  pdfDoc
    .createPage('letter-size')
    .image(IMAGE)
    .endPage()
    .endPDF();
}
module.exports = CreateSimplePdf;
```

```javascript
// index.js
/*
note that index.js is placed in the root of the app, 
and generate Pdf.js is placed inside a folder called service
*/
const generatePdf = require('./service/generatePdf.js');
generatePdf();
```
if run: `node index.js` will throw `TypeError: Unable to end PDF` because cannot find the image that is placed service, beside generatePdf.js i don't know why sometimes throw `Segmentation fault (core dumped)`

simple fix: 

```javascript
//fix #1
// service/generatePdf.js
const IMAGE = './service/image.jpg';//put the path to the image like it was placed in the root of the app
```

```javascript
//fix #2
// service/generatePdf.js
const path = require('path');
const IMAGE =  path.resolve(__dirname, './image.jpg');//use path module to resolve the path
```

**Comment by chunyenHuang** [comment]
Good to know. Thanks
